ui/admin/tests/acceptance/credential-stores-test.js

```diff
@@ -236,4 +238,54 @@ module('Acceptance | credential-stores', function (hooks) {
       );
     }
   });
+
+  test('can delete credential store', async function (assert) {
+    assert.expect(1);
+    const count = getCredentialStoresCount();
+    await visit(urls.credentialStore);
+    await click('.rose-layout-page-actions .rose-dropdown-button-danger');
+    assert.equal(getCredentialStoresCount(), count - 1);
+  });
+
+  test('can accept delete credential store via dialog', async function (assert) {
+    assert.expect(2);
+    const confirmService = this.owner.lookup('service:confirm');
+    confirmService.enabled = true;
+    confirmService.confirm = sinon.fake.returns(resolve());
+    const count = getCredentialStoresCount();
+    await visit(urls.credentialStore);
+    await click('.rose-layout-page-actions .rose-dropdown-button-danger');
+    assert.equal(getCredentialStoresCount(), count - 1);
+    assert.ok(confirmService.confirm.calledOnce);
+  });
+
+  test('cannot cancel delete credential store via dialog', async function (assert) {
+    assert.expect(2);
+    const confirmService = this.owner.lookup('service:confirm');
+    confirmService.enabled = true;
+    confirmService.confirm = sinon.fake.returns(reject());
+    const count = getCredentialStoresCount();
+    await visit(urls.credentialStore);
+    await click('.rose-layout-page-actions .rose-dropdown-button-danger');
+    assert.equal(getCredentialStoresCount(), count);
+    assert.ok(confirmService.confirm.calledOnce);
+  });
+
+  test('deleting a host catalog which errors displays error messages', async function (assert) {
```

Comment:
`host catalog`?

Reply:
:man_facepalming: my mind didn't pay much attention to that one... FIX!